### DISCUSSION TITLE
Looking for early WIP testers - prior to the next Dev release

### DISCUSSION THREAD

**Discussion by BigBobbas** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#58] [Announcements]
Hi Guys, 
I have added some new features (please bare in mind that this config is not ready for general use and is not complete - this may not include any recent PR's that have already been merged)
[HERES THE LINK TO THE CONFIG](<https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/blob/dev-live-wip/s3b_dev.yaml>)
I have included the following and would love someone to test to see if functionality is as expected.

Release Notes: 

1. Added uptime calculations to display days, months, minutes, seconds

2. Changed entity icons in HA Integration and tried to tidy the best it allows. NOT completed fully

3. Added Timeout settings for screensaver and blank screen Please note that Screensaver must be on in order for blank screen to be enabled.
   You can now adjust the timeout delays from settings>>saver>>timeout settings page.
   
4. Switch media player to external player - This can now be done from the device in settings>> voice settings without editing the config. Add media player entity in the substitutions at the top of the config. 
***** I HAVE NOT ADDED THE SWITCH ON THE DEVICE YET, BUT IT IS AVAILABLE IN THE HA INTEGRATION - ALSO I HAVE NOT ADDED THE WAKE SOUND YET - I WANTED TO GET THIS TESTED BEFORE WORKING FURTHER ON IT *****

As always thank you so much for your continued support with this project. The only reward that I need or want from doing this is knowing that you amazing people are loving it as much as I am making it.

Best Wishes 

Rob

**Comment by gadicc** [comment]
Ok, I didn't try `dev-live-wip` but just installed the latest `s3b_dev.yaml` that's merged to `dev` and all I can say is you're friggin' awesome and this project is friggin' awesome! 🤩  Amazing to see what you've accomplished in 3 months and super excited to watch this project continue to mature :clap: :clap: :clap:

**Comment by mikey60** [comment]
I have been running the s3b_dev.yaml and s3b_dev_no_sensors.yaml of the https://github.com/BigBobbas/ESP32-S3-Box3-Custom-ESPHome/tree/dev-live-wip branch.  I am running Home Assistant using Duck DNS addon (https).  I also have disabled "Output audio externally" so I am playing over the internal speaker.  My early findings are as follows:
When playing music and using the media screen:
- The volume -/+ and mute controls work as expected. 
- The play/pause button stops the music playing and clears the queue.
- Nothing playing is shown even though music is playing

One minor thing I also noticed is that when updating the saver screen brightness,  the updated brightness is not correct until the screen is refreshed.

**Reply by BigBobbas** [reply]
Hi,
thank you for your feedback, for testing and your setup info. 
I will have a look into these points.
I have fixed the brightness updating but not uploaded changes yet (you reminded me lol )
Many Thanks and Best Wishes
Rob 👍

**Comment by Cadair** [comment]
Hi, I am trying out the s3b_dev.yaml config and it's working well but I can't get the wakeup and timer sounds to play. I have added them to hass, and through the media library UI in hass I can play them to the media_player on the s3, however, when doing that there is an auth token in the URL which isn't in the config, so I assume when the s3 is trying to play them it's having an auth issue?

**Reply by BigBobbas** [reply]
Hi, if you add them to HA >>config/www/sounds  (if sounds folder doesn’t exist then create it )
also make sure that the device is allowed to make service calls – in HA >> device and services >> esphome>> click ‘configure’ next to the device name. and make sure the option is ticked
